.github/workflows/sdk-execution.yml

```diff
@@ -33,43 +33,14 @@ jobs:
         with:
           k8s_version: ${{ matrix.k8s_version }}
 
-      - name: Forward API port
-        run: ./.github/resources/scripts/forward-port.sh "kubeflow" "ml-pipeline" 8888 8888
-
-      - name: apt-get update
-        run: sudo apt-get update
-
-      - name: Install protobuf-compiler
-        run: sudo apt-get install protobuf-compiler -y
-
-      - name: Install setuptools
-        run: |
-          pip3 install setuptools
-          pip3 freeze
-
-      - name: Install Wheel
-        run: pip3 install wheel==0.42.0
-
-      - name: Install protobuf
-        run: pip3 install protobuf==4.25.3
-
-      - name: Generate API proto files
-        working-directory: ./api
-        run: make clean python
-
-      - name: Install kfp-pipeline-spec from source
-        run: |
-          python3 -m pip install api/v2alpha1/python
-
       - name: Install requirements
-        run: pip install -r ./test/sdk-execution-tests/requirements.txt
+        run: make setup-sdk-execution
```

Comment:
this looks like we are deploying KFP twice [[1]](https://github.com/kubeflow/pipelines/pull/11509/files#diff-c69071fbcb3d7349e45c6575c3516466be788ea3caa600bf653529f992c553edR31), [[2]](https://github.com/kubeflow/pipelines/pull/11509/files#diff-b224dfdea0489e673f571b5ff2c821aadf154a630918adac30e0258ce73f7ecaR146) ?